app/src/main/java/co/nimblehq/compose/crypto/ui/theme/Dimension.kt

```diff
@@ -20,4 +20,9 @@ object Dimension {
     val Dp45 = 45.dp
     val Dp52 = 52.dp
     val Dp60 = 60.dp
+
+    // Card shadow
+    val shadowBorderRadius = 12.dp
+    val shadowBlurRadius = 24.dp
+    val shadowOffsetY = 212.dp
```

Comment:
@hoangnguyen92dn Here's the result on my device `Samsung Galaxy S10+` 
![Screenshot_20221018_103622](https://user-images.githubusercontent.com/28002315/196329993-24357067-fe41-4629-9292-7c4182364bcd.png). The shadow is very faded 🤔 Not sure It's because we set this `height` or not

Reply:
@Wadeewee You're right, the `shadowOffsetY` determines the position of shadow box on Y axis so it would change on different devices. Let me have more insight about this.

Reply:
@Wadeewee It took time to support different device specifications for now. I think we could go with this approach currently.

Reply:
@hoangnguyen92dn I think the issue is about the `dimension,` but yeah, we can fix it later 🚀  cc @luongvo